content/shmem_fence.tex

```diff
@@ -43,19 +43,29 @@
     ordering of its memory accesses.
     There is a subtle difference between
     \FUNC{shmem\_fence} and \FUNC{shmem\_quiet}, in that, \FUNC{shmem\_quiet}
-    guarantees completion of \PUT{}, \acp{AMO}, and memory store routines to
+    guarantees completion of \PUT{}, \ac{AMO}, memory store, and non-blocking \PUT{} routines to
     symmetric data objects which makes the updates visible to all other
     \acp{PE}. 
     
     The \FUNC{shmem\_quiet} routine should be called if completion of \PUT{},
-    \acp{AMO}, and memory store routines to symmetric data objects is desired
+    \ac{AMO}, memory store, and non-blocking \PUT{} routines to symmetric data objects is desired
     when multiple remote \acp{PE} are involved.
+
+    In an \openshmem program with multithreaded \acp{PE}, it is the
```

Comment:
@manjugv Did you have time to review this piece of text?

Reply:
i have closed the issue based on Manju's feedback